components/dot_menu/dot_menu.tsx

```diff
@@ -336,12 +336,17 @@ export default class DotMenu extends React.PureComponent<Props, State> {
             }) || [];
 
         const appsBindings = this.props.appsBindings.map((item) => {
+            let icon: JSX.Element | undefined;
+            if (item.icon) {
```

Comment:
if this is the case, I am 1/5 inclined to rename the field iconurl. We need to decide on restrictions we apply to static assets - should probably be served out of the rooturl sub-URL, or the marketplace.

Reply:
This is the case now, and I think it is where we should move forward (easy, allow both internal and external files, and since it is the img src, it should be "safe"), but I think we haven't discussed yet what we want here. Also, we should document what format do we allow here. At the moment would be "anything that can be interpreted by the `<img>` html tag.

Reply:
Filed https://mattermost.atlassian.net/browse/MM-30595